assets/css/access.css

```diff
@@ -66,7 +66,7 @@
     font-family: 'Source Sans Pro',sans-serif;
 }
 
-.button:hover {
+.button:hover:enabled {
```

Comment:
I would say that is acceptable, we only fully support >= IE9 on theguardian.com so I think the same standard is OK on open platform site.